CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+
+# Change Log
+All notable changes to this project will be documented in this file.
+ 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+ 
+## [Unreleased] - yyyy-mm-dd
+ 
+Here we write upgrading notes for brands. It's a team effort to make them as
+straightforward as possible.
+ 
+### Added
+- 13-07-2022 [ROSbot2-#2kpq27q](https://app.clickup.com/t/2kpq27q)
```

Comment:
Iiiiiii... Don't think pointing to Cickup task on public repo is a good idea... Apart from that, it's fine.

Reply:
You're right, I forget that this repo is public :D

Reply:
:heavy_check_mark: